packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/cancel-resiliation/cancel-resiliation.module.js

```diff
@@ -2,10 +2,14 @@ import angular from 'angular';
 import '@uirouter/angularjs';
 import ovhManagerBilling from '@ovh-ux/manager-billing';
 
+import billingAutorenewService from '../../../../billing/autoRenew/autorenew.service';
 import routing from './cancel-resiliation.routing';
 
 const moduleName = 'ovhManagerBillingAutorenewCancelResiliation';
 
-angular.module(moduleName, ['ui.router', ovhManagerBilling]).config(routing);
+angular
+  .module(moduleName, ['ui.router', ovhManagerBilling])
+  .config(routing)
+  .service('BillingAutoRenew', billingAutorenewService);
```

Comment:
Can't we have any side effects by declaring the service twice ?

Reply:
As discussed, to make sure that there are no side-effects, I have given a new name to the service definition...